### DISCUSSION TITLE
How to set up a development environment with Phoenixd?

### DISCUSSION THREAD

**Discussion by brh28** [ACINQ/phoenixd#58] [Q&A]
I'm looking to integrate Phoenixd to a web application I run. Setting up a development environment for other node implementations is pretty easy using a tool like Polar. But to my understanding, Polar doesn't support Phoenixd and manually configuring a regtest would be difficult (if not impossible), since phoenixd has to connect to the Acinq node. 

Some alternative options I see:
1. Run a testnet phoenixd. The problem here is how to fund the testnet channel? I don't see any lightning faucets online which means I would have to create a testnet node using a different lightning implementation, open a channel, and then send funds to my phoenixd node. Perhaps doable, but a lot of friction.
2. Test on mainnet. Not ideal of course, but it's a low lift. I'd prefer only sending small amounts and never opening a channel, but it appears you cannot spend without a channel 
```
./phoenix-cli payinvoice --invoice=<lnbc1...>
{
    "paymentHash": "...",
    "reason": "payment could not be sent through existing channels, check individual failures for more details"
}
```

If you have thoughts, would be happy to hear em.

**Comment by pm47** [comment]
We recommend testnet. There are few faucets (e.g. https://bitcoinfaucet.uo1.net/), and you can always ping us with your lightning offer and we'll send some funds.
